bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/ConicDetailEditor.scala

```diff
@@ -0,0 +1,132 @@
+package jsky.app.ot.gemini.editor.targetComponent.details
+
+import edu.gemini.pot.sp.ISPNode
+import edu.gemini.shared.util.immutable.{ Option => GOption }
+import edu.gemini.spModel.obs.context.ObsContext
+import edu.gemini.spModel.target.SPTarget
```

Comment:
Would it make sense to make the new editors work with the new Target model classes?